docs/devs/e2e-tests.md

```diff
@@ -86,13 +90,13 @@ test.afterEach(async ({ page }) => {
 });
 ```
 
-**Simplified Explanation**:
+**Simplified Explanation**
 
 **Test Setup**: Before each test, login to O3, create a new patient, and start a visit.
 
 **Test Case**:
 - **Setup**: Navigate to the lab order form, add a new lab order, and save it.
-- **Replay**: Go to the SENAITE application and search for the client.
+- **Reply**: Go to the SENAITE application and search for the client.
```

Comment:
Should be Replay?

Reply:
@Ruhanga as well should it be `setup` or `set up`?

Reply:
Actually should be `setup`